ext/zend_test/test.stub.php

```diff
@@ -51,6 +51,7 @@ trait _ZendTestTrait {
         /** @var mixed */
         public $testProp;
 
+        #[Attribute(Attribute::TARGET_METHOD)]
```

Comment:
The `Attribute` attribute is not appropriate for a method (it's a class attribute). Please apply `ZendTestAttribute` instead (or `ReturnTypeWillChange`).